src/asmcnc/apps/systemTools_app/screens/xy_jig/screen_xy_jig.py

```diff
@@ -6,10 +6,19 @@
 from asmcnc.skavaUI import popup_info
 from asmcnc.apps.systemTools_app.screens.xy_jig.popup_xy_jig import *
 
-import matplotlib
-matplotlib.use('Agg')
-import matplotlib.pyplot as plt
-import matplotlib.ticker as plticker
+from datetime import datetime
+
+def log(message):
+    timestamp = datetime.now()
+    print (timestamp.strftime('%H:%M:%S.%f' )[:12] + ' ' + message)
+
+try:
+    import matplotlib
+    matplotlib.use('Agg')
+    import matplotlib.pyplot as plt
+    import matplotlib.ticker as plticker
+except Exception as e:
+    print(str(e))
```

Comment:
did you intend to log this rather than print? (only bc you added the log function above it)

Reply:
Whoops...